src/collections/_documentation/development/sdk-dev/unified-api.md

```diff
@@ -54,7 +54,20 @@ meant that certain integrations (such as breadcrumbs) were often not possible.
 
 - **integration**: Code that provides middlewares, bindings or hooks into certain frameworks or environments, along with code that inserts those bindings and activates them. Usage for integrations does not follow a common interface.
 
-- **event processors:** Are callbacks that run for every event. They can either return a "new" event which in most cases means just adding data OR return `null` in case the event will be dropped and not sent.
+- **event processors**: Callbacks that run for every event.  
+  They can either return a "new" event, which in most cases means just adding
+  data OR return `null` in which case the event will be discarded and not
+  processed further.
+
+  See [Event Pipeline](#event-pipeline) for more information.
+
+- **disabled SDK**: Most of the SDK functionality depends on a
+  configured and active Client.
+  Sentry considers the client active when it has a *transport*.
+  Otherwise, the client is inactive, and the SDK is considered "disabled".
+  This means that certain callbacks, such as `configure_scope` or
```

Comment:
Suggest "When a client is inactive and the SDK is considered disabled, certain callbacks, such as `configure_scope` or *event processors*, may not be invoked. As a result, breadcrumbs are not recorded.